utilities/decrypt_data/export_postgres_to_excel_chunks.py

```
@@ -0,0 +1,585 @@
+            
+        except Exception as e:
+            error_msg = f"Error exporting table {schema}.{table_name}: {e}"
+            logger.error(error_msg)
+            result['error'] = str(e)
+            try:
+                self.connection.rollback()
+            except Exception:
+                pass
+            return result
+    
+    def export_table_to_excel(self, table_name: str, schema: str, output_dir: str = 'exported_tables', use_chunking: bool = True, chunk_size: int = 10000) -> dict:
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)